RealmSwift/Tests/MigrationTests.swift

```diff
@@ -170,33 +174,35 @@ class MigrationTests: TestCase {
     }
 
     func testCreate() {
+        autoreleasepool {
+            _ = Realm()
+        }
+
         migrateAndTestRealm(Realm.defaultPath, block: { migration, oldSchemaVersion in
             migration.create("SwiftStringObject", value: ["string"])
             migration.create("SwiftStringObject", value: ["stringCol": "string"])
             migration.create("SwiftStringObject")
 
             self.assertThrows(migration.create("NoSuchObject", value: []))
-
-            var count = 0
-            migration.enumerate("SwiftStringObject", { oldObj, newObj in
-                if count == 0 {
-                    // first object has default value of empty string
-                    XCTAssertEqual(newObj!["stringCol"] as String, "")
-                }
-                else {
-                    XCTAssertEqual(newObj!["stringCol"] as String, "string")
-                    XCTAssertNil(oldObj, "Objects created during migration have nil oldObj")
-                }
-                count++
-            })
-            XCTAssertEqual(count, 3)
         })
 
         XCTAssertEqual(Realm().objects(SwiftStringObject.self).count, 3)
+
+        var i = 0
+        for obj in Realm().objects(SwiftStringObject) {
```

Comment:
Since we only have once conditional for each element might be less code if we just remove the loop and have the 3 `XCTAssertEqual`s